src/domains/queries.ml

```diff
@@ -61,6 +61,7 @@ type partaccess = {exp: CilType.Exp.t; var_opt: CilType.Varinfo.t option; write:
 
 (** GADT for queries with specific result type. *)
 type _ t =
+  | Assert: exp -> ES.t t
```

Comment:
If this stays at all (see above comment), then it should go last, just like it's number in `equal` is. Although even then the name is not very helpful because it's more like `Invariant`, although then it feels completely redundant.